### PR TITLE
**Breaking:** Remove deprecated constants

### PR DESCRIPTION
**Pull request by TejasQ**
# ⚠️ THIS PR IS VERY LIKELY TO HAVE VISUAL REGRESSIONS ⚠️ 

PLEASE @micha-f and @contiamo/frontend, spend a few minutes and go through each component to see which sizes/paddings/etc. broke. 

## The components whose snapshots changed need extra attention.

It may be helpful to have (side by side) the [PR preview](https://deploy-preview-929--operational-ui.netlify.com/) and [`master`](https://operational-ui.netlify.com). I am doing the same.

This PR removes all of the unnecessary dead weight in the library.

# Related issue

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

File: src/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -20,8 +20,8 @@ const Container = styled("div")({
 
 const Slash = styled("span")(({ theme }) => ({
   display: "inline-block",
-  margin: `0 ${theme.deprecated.spacing / 2}px`,
-  color: theme.deprecated.colors.gray,
+  margin: theme.space.medium,
```
Comment (TejasQ) [inline src/Breadcrumbs/Breadcrumbs.tsx]:
```suggestion
  margin: `0 ${theme.space.medium}px`,
```

**Comment by fabien0102** [comment]
image

This example is broken, even on master, let's remove the old API 😅

**Comment by fabien0102** [comment]
image

This should be green

**Comment by fabien0102** [comment]
image

The toggle switch is broken

**Comment by micha-f** [comment]
I think we need a thorough review of all components before we can merge this. @fabien0102 did you go through all components? Or is your feedback just the result of a quick check?

**Pull request merged**